README.md

```diff
@@ -1,4 +1,40 @@
-bioformats-examples
-===================
+Bio-Formats examples
+====================
 
-Example code that demonstrates the Bio-Formats API in Java and C++
+This repository contains minimal examples of usage of Bio-Formats as a Java
+library using either Maven or Gradle.
+
+Prerequisites
+-------------
+
+The following are required:
+
+    JDK 7 or higher
+    Maven 2.x (for Maven build)
+
+Building with Maven
+-------------------
+
+From the root directory, run:
+
+    mvn
```

Comment:
:+1: to keep the instructions minimal, f6edfde7baabe1b365ffcfed271df8938893bf9f adds a default goal running the `clean` and `install` phases.